src/main/kotlin/org/sd_jwt/SdJwt.kt

```diff
@@ -316,22 +417,59 @@ fun checkDisclosuresMatchingDigest(sdJwt: JSONObject, disclosureMap: HashMap<Str
  */
 inline fun <reified T> createPresentation(
     credential: String,
+    claims: T,
```

Comment:
Do we really need this parameter here? In this case we don't need to check the correctness of the JSON structure and I don't want to change the function interface for people already using this library.
My suggestion would be:
1. Remove the JSON structure test and rename your function below to internalCreatePresentation
2. Create a new createPresentation function to process JSONObjects and check the structure
3. In the new function call the internalCreatePresentation function
4. From this function call the internalCreatePresentation function

Reply:
thank you fo the suggestion, I will change the PR accordingly